README.md

```diff
@@ -98,6 +98,20 @@ Convert an observation to an OSM type `node`.
 
 Retrieves the current device's unique ID (string).
 
+### `status = mapeo.getDatabaseStatus()`
+
+Retrieves the status of the database, returns an array of hypercore feed
```

Comment:
```suggestion
Retrieves the status of the database, returns an array of Device IDs and their download status.
```

Reply:
I was thinking (as per discussions /w @gmaclennan) we could try and use higher-level terminology in the mapeo-* modules, and do our best to keep hypercore-y terminology under the API covers. Does this sound ok with you?